src/anyvar/restapi/schema.py

```diff
@@ -0,0 +1,190 @@
+"""Provide response definitions to REST API endpoint."""
+from enum import Enum
+from typing import Any, Dict, List, Optional, Type, Union
+
+from ga4gh.vrsatile.pydantic.vrs_models import Allele, SequenceLocation, Text
+from pydantic import BaseModel, StrictInt, StrictStr
+
+
+class EndpointTag(str, Enum):
+    """Denote endpoint group membership"""
+
+    GENERAL = "General"
+    SEQUENCES = "Sequences"
+    LOCATIONS = "Locations"
+    VARIATIONS = "Variations"
+    SEARCH = "Search"
+
+
+class DependencyInfo(BaseModel):
+    """Provide information for a specific dependency """
+
+    version: StrictStr
+
+
+class InfoResponse(BaseModel):
+    """Describe response for the /info endpoint"""
+
+    anyvar: DependencyInfo
+    ga4gh_vrs: DependencyInfo
+
+    class Config:
+        """Configure InfoResponse class"""
+
+        @staticmethod
+        def schema_extra(schema: Dict[str, Any], model: Type["InfoResponse"]) -> None:
+            """Configure OpenAPI schema"""
+            if "title" in schema.keys():
+                schema.pop("title", None)
+            for prop in schema.get("properties", {}).values():
+                prop.pop("title", None)
+            schema["example"] = {
+                "anyvar": {
+                    "version": "0.1.2.dev58+g81eb592.d20230316"
+                },
+                "ga4gh_vrs": {
+                    "version": "0.7.6"
+                }
+            }
+
+
+class GetSequenceLocationResponse(BaseModel):
+    """Describe response for the /locations/ endpoint"""
+
+    location: SequenceLocation
+
+
+class RegisterVariationRequest(BaseModel):
+    """Describe request structure for variation registration endpoint"""
+
+    definition: StrictStr
+    untranslatable_to_text: bool = False
```

Comment:
I have some sort of vague questions around how to handle untranslatable variation descriptions, but I figured this parameter was an easy stopgap until we settle things. I guess I'm not sure if there are specific *reasons* for a failure to normalize that we'd like to accept or reject (e.g., maybe an invalid nucleotide should be rejected entirely, but something that we can't tokenize would be something we'd want to keep just to be safe?)